py3dtilers/CityTiler/README.md

```diff
@@ -195,3 +195,25 @@ Read the texture from the OBJ and write it in the produced 3DTiles:
 ```bash
 citygml-tiler --db_config_path <path_to_file>/Config.yml --with_texture
 ```
+
+### Geometric error
+
+In 3DTiles, [the geometric error](https://github.com/CesiumGS/3d-tiles/tree/main/specification#geometric-error) is the metric used to refine a tile or not. A tile should always have a lower geometric error than its parent. The root of the tileset should have the highest geometric error and the leaves the lowest geometric error.
+
+The geometric errors of each "type" of tiles (basic, LOD1 or LOA) can be overwrited with the flag `--geometric_error`. The values after the flag will be used (from left to right) for basic tiles, LOD1 tiles and LOA tiles.
+
+```bash
+citygml-tiler --db_config_path <path_to_file>/Config.yml --geometric_error 5 60 100  # Set basic tiles GE to 5, LOD1 tiles GE to 60 and LOA tiles GE to 100
```

Comment:
Define GE in parenthesis for its couple first encounters e.g. "...LOA tiles GE to..." to become "...LOAD tiles GE (Geometric Error) to ..."